nifi-api/src/main/java/org/apache/nifi/authorization/AccessPolicy.java

```diff
@@ -28,15 +28,15 @@
 
     private final String identifier;
```

Comment:
@bbende @mcgilman We discussed adding a description field to AccessPolicy and Group.  I created the following JIRA ticket for that effort: https://issues.apache.org/jira/browse/NIFI-1949